_docs/vision.md

```diff
@@ -59,5 +59,4 @@ people improve the user experience, raising it to new levels.
 
```

Comment:
I do think the whole first paragraph could go... the second paragraph says what you need to say.

Reply:
Is f47c4bf7d90bc92030bbcbf125f20a95ecc03c93 what you had in mind?

I kind of liked that paragraph... this is not documentation, this is motivational speech.
I changed the heading for the section, since it no longer matches.

Reply:
I agree with Susan - we should avoid unnecessary text. Users who got this far will be motivated anyway.

Reply:
@rcv-aws I have already removed the paragraph in the above commit.